README.md

```diff
@@ -34,8 +38,6 @@ bash scripts/patch_openmm.sh
 You can get the FastFold source and install it with setuptools:
```

Comment:
Maybe till this line, I have got the source code?

Reply:
fixed